modules/core/src/v2/wallet.ts

```diff
@@ -105,6 +105,8 @@ export interface GetUserPrvOptions {
   walletPassphrase?: string;
 }
 
+type ManageUnspents = 'consolidate' | 'fanout';
```

Comment:
export this type, unless you elect to make manageUnspents private as commented below.

Reply:
will make private